src/steps/os/openbsd.rs

```diff
@@ -4,30 +4,36 @@ use crate::terminal::print_separator;
 use crate::utils::{get_require_sudo_string, require_option};
 use color_eyre::eyre::Result;
 use rust_i18n::t;
+use std::fs;
 
 fn is_openbsd_current(ctx: &ExecutionContext) -> Result<bool> {
+    let motd_content = fs::read_to_string("/etc/motd")?;
```

Comment:
Reading a file in a dry run harms nothing! Nice!

Reply:
I cant tell if this is genuine or sarcastic.

Reply:
Emm, I seriously mean this is good, I think we can do anything in a dry run as long as we don't do the update. 😁

Reply:
gotcha, as you may have noticed, I ain't a great rust dev, so I couldn't tell if you were playfully mocking me.

thanks for clarifying.